core/models.py

```diff
@@ -1207,12 +1204,18 @@ def __unicode__(self):
         return "path=%s size=%s sha1=%s" % (self.path, self.size, self.sha1)
 
     def _add_page(self, page, tar):
+        from index import get_page_text
+
         d = page.issue.date_issued
         relative_dir = "%s/%i/%02i/%02i/ed-%i/seq-%i/" % (page.issue.title_id, d.year, d.month, d.day, page.issue.edition, page.sequence)
 
         # add ocr text
         txt_filename = relative_dir + "ocr.txt"
-        ocr_text = page.ocr.text.encode('utf-8')
+        ocr_text = get_page_text(page)
+        if len(ocr_text) > 0:
```

Comment:
This can simply be `if ocr_text:`

Reply:
Now reverted to ocr_text = get_page_text(page)[0].encode('utf-8')